coreneuron/apps/main1.cpp

```diff
@@ -191,6 +191,12 @@ void nrn_init_and_load_data(int argc,
                 "missing --gpu\n");
         exit(1);
     }
+    if (!corenrn_param.gpu && corenrn_param.cuda_interface) {
+        fprintf(stderr,
+                "compiled with OpenACC/CUDA does not allow the combination of --cuda-interface and "
+                "missing --gpu\n");
+        exit(1);
+    }
```

Comment:
I'm surprised we're not hitting this, as the new `cudaInterface` tests seem to be running in non-GPU builds (that do not pass `--gpu`).